swagger.yaml

```diff
@@ -146,226 +328,821 @@ paths:
+          $ref: '#/responses/InternalError'
   /datasets/{id}/editions/{edition}/versions/{version}/dimensions/{dimension}:
     get:
       tags:
-      - "Dataset"
+      - "Public"
       summary: "Get all values from a dimension"
       description: "Get a list of all values which appear in this dimension and dataset"
       parameters:
-      - $ref: '#/parameters/id'
+      - $ref: '#/parameters/dimension'
       - $ref: '#/parameters/edition'
+      - $ref: '#/parameters/id'
       - $ref: '#/parameters/version'
-      - $ref: '#/parameters/dimension'
       responses:
         200:
           description: "Json object containing all values for a dimension"
           schema:
             $ref: '#/definitions/DimensionValues'
         404:
-          description: "No dataset edition with the specific version was found using the id, edition and version provided or the dimension does not exist within the dataset"
+          description: "No version was found for an edition of a dataset using the id, edition and version provided or the dimension does not exist within the dataset"
         500:
-          description: "Failed to process the request due to an internal error"
+          $ref: '#/responses/InternalError'
+  /instances:
+    get:
+      tags:
+      - "Private user"
+      summary: "Get a list of instances"
+      description: "Get a list of instances which has been paged"
+      parameters:
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+        - $ref: '#/parameters/state'
+      produces:
+      - "application/json"
+      security:
+      - FlorenceAPIKey: []
+      responses:
+        200:
+          description: "Return a list of instance state"
+          schema:
+            $ref: '#/definitions/Instances'
+        500:
+          $ref: '#/responses/InternalError'
+    post:
+      tags:
+      - "Private"
+      summary: "Create an instance"
+      description: "Create an instance which will be imported"
+      parameters:
+      - $ref: '#/parameters/import_job'
+      produces:
+      - "application/json"
+      security:
+      - InternalAPIKey: []
+      responses:
+        200:
+          description: "Create an instance then returned it"
+          schema:
+            $ref: '#/definitions/Instance'
+        500:
+          $ref: '#/responses/InternalError'
+  /instances/{instance_id}:
+    get:
+      tags:
+      - "Private user"
+      summary: "Get Information about an instance"
+      description: "Get the current state of an instance, this includes all events which have happened."
+      parameters:
+      - $ref: '#/parameters/instance_id'
+      produces:
+      - "application/json"
+      security:
+      - FlorenceAPIKey: []
+      responses:
+        200:
+          description: "Return a single instance state"
+          schema:
+            $ref: '#/definitions/Instance'
+        404:
+          description: "InstanceId does not match any import jobs"
```

Comment:
Should this now be something like "InstanceId does not match any instances"?

Reply:
done